### ISSUE TITLE
message: The "RtCli" task was not given a value for the required parameter TargetFramework

### ISSUE THREAD

**Issue by mcshaz** [reinforced/Reinforced.Typings#72]
Thank you for great bit of work

I had to change build\Reinforced.Typings.targets from

    <RtCli 
      TargetFramework="$(TargetFramework)"

to get the project to build

    <RtCli 
      TargetFramework="net461"

maybe because I am using .net 4.6.2. I am not sure the best solution to make it build automatically, but at the very least something in the docs about how to fix this error

**Comment by zivillian** [comment]
I had the same problem and just added 
``` xml
<PropertyGroup>
+  <TargetFramework>net461</TargetFramework>
   <RT...
```
to my Reinforced.Typings.settings.xml

**Comment by abritabroad** [comment]
> I had the same problem and just added
> 
> ```
> <PropertyGroup>
> +  <TargetFramework>net461</TargetFramework>
>    <RT...
> ```
> 
> to my Reinforced.Typings.settings.xml

Cheers - you saved my life.... I couldn't figure out why my typings were not updating... This needs to be in the Read.ME

**Comment by pavel-b-novikov** [comment]
It should be fixed, not added to readme :(

**Comment by pavel-b-novikov** [comment]
Fixed in 1.4.97 release by adding [RtForceTargetFramework](https://github.com/reinforced/Reinforced.Typings/wiki/Reinforced.Typings.settings.xml#RtForceTargetFramework)
